src/guides/v2.3/extension-dev-guide/view-models.md

```diff
@@ -133,9 +133,10 @@ The following is an example of the view model initialization in the `app/code/Ma
 
 ```php
 /** @var $viewModel /Magento/Catalog/ViewModel/Product/Listing/PreparePostData */
+/** @var $escaper-> \Magento\Framework\Escaper */
```

Comment:
Should this not be -
```suggestion
/** @var $escaper \Magento\Framework\Escaper */
```

Reply:
@BarnyShergold Your suggested changes are done.